packages/relay/src/lib/eth.ts

```diff
@@ -1553,17 +1561,22 @@ export class EthImpl implements Eth {
    */
   async sendRawTransaction(transaction: string, requestDetails: RequestDetails): Promise<string | JsonRpcError> {
     const requestIdPrefix = requestDetails.formattedRequestId;
-    if (transaction?.length >= constants.FUNCTION_SELECTOR_CHAR_LENGTH)
```

Comment:
I see this increment of the log has been moved to line 1569 but didn't understand why we removed this check

Reply:
removing the checks, because we are adding more labels to the existing counter. the checks were for contract call and creation. now we want the counter to be incremented on a method call and depending on the labels we'll filter it in grafana.

Reply:
Ah so earlier we want to only capture this metric for contract call which has a calldata > than constants.FUNCTION_SELECTOR_CHAR_LENGTH (basically the 4 bytes selectors). But now we can reuse the execution counter to capture metric in different methods. Okay good point! Thanks for confirming!